jobs/wclouser-fxa-db-counts/wclouser_fxa_db_counts/main.py

```diff
@@ -0,0 +1,135 @@
+            UNION ALL
+        SELECT 'totp' AS table_name, COUNT(*) AS total_rows FROM `moz-fx-data-shared-prod.accounts_db_external.fxa_totp_v1`
+            UNION ALL
+        SELECT 'unblock_codes' AS table_name, COUNT(*) AS total_rows FROM `moz-fx-data-shared-prod.accounts_db_external.fxa_unblock_codes_v1`
+            UNION ALL
+        SELECT 'unverified_tokens' AS table_name, COUNT(*) AS total_rows FROM `moz-fx-data-shared-prod.accounts_db_external.fxa_unverified_tokens_v1`
+            UNION ALL
+        SELECT 'verification_reminders' AS table_name, COUNT(*) AS total_rows FROM `moz-fx-data-shared-prod.accounts_db_external.fxa_verification_reminders_v1`
+            UNION ALL
+        (SELECT
+           "accounts_with_secondary_emails" AS table_name,
+            COUNT(DISTINCT accounts.uid) AS total_rows
+        FROM
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_accounts_v1` accounts
+        JOIN
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_emails_v1` emails
+        ON
+            accounts.uid = emails.uid
+        WHERE
+            emails.isPrimary = FALSE
+        )
+            UNION ALL
+        (SELECT
+            "accounts_with_unverified_emails" AS table_name,
+            COUNT(DISTINCT accounts.uid) AS total_rows
+        FROM
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_accounts_v1` accounts
+        JOIN
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_emails_v1` emails
+        ON
+            accounts.uid = emails.uid
+        WHERE
+            emails.isVerified = FALSE
+        )
+            UNION ALL
+        (
+        SELECT
+            "accounts_linked_to_google" AS table_name,
+            COUNT(uid) AS total_rows
+        FROM
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_linked_accounts_v1`
+        WHERE
+            providerId=1 -- see LinkedAccountProviderIds at https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/lib/types.ts
+        )
+            UNION ALL
+        (
+        SELECT
+            "accounts_linked_to_apple" AS table_name,
+            COUNT(uid) AS total_rows
+        FROM
+            `moz-fx-data-shared-prod.accounts_db_external.fxa_linked_accounts_v1`
+        WHERE
+            providerId=2 -- see LinkedAccountProviderIds at https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/lib/types.ts
+        )
+    """
+    
+    client = bigquery.Client(project="moz-fx-fxa-prod")
+    query_job = client.query(query)
+
+    print("Running query:", query_job.job_id)
+
+    results = query_job.result()
+
+    rows_to_insert = [
+        {
+            "date": datetime.now().strftime("%Y-%m-%d"),
+            "name": row["table_name"],
+            "count": row["total_rows"],
+        }
+        for row in results
+    ]
+
+    # Insert rows into the new table
+    errors = client.insert_rows_json("mozdata.analysis.wclouser_fxa_health_db_counts", rows_to_insert)
```

Comment:
This is fine in an MVP, but we can also parameterize this to put this data in `accounts_backend_derived`.
Note that tables created in `analysis` dataset now have default expiration of 6 months which can be changed manually.